### PR TITLE
Dev/tuminer

### PR DESCRIPTION
**Pull request by monsieurzhang**

### CODE REVIEW

File: utilities/tuminer/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ usage: entrypoint.py [-h] [-s STORAGE_CONFIG] [-t TASK_ID] [-i IMAGE]
                      [-b BEAT_URL] [-bi BEAT_INTERVAL]
```
Comment (guillaumekln) [inline utilities/tuminer/README.md]: Can you take the opportunity to remove the section "Command line options"? It duplicates the code.

File: utilities/tuminer/README.md [hunk 2]
```diff
@@ -62,8 +62,8 @@ optional arguments:
 nvidia-docker run -it \
   -v $PWD/test/corpus:/root/corpus:/corpus \
   -v /tmp/output:/output \
-  opennmt/tuminer \
-  --mode score \
+  nmtwizard/tuminer \
```
Comment (guillaumekln) [inline utilities/tuminer/README.md]: Should it be `nmtwizard/tuminer tuminer` instead?

Comment (guillaumekln) [reply utilities/tuminer/README.md]: Ok this is fixed.

File: utilities/tuminer/README.md (hunk 1 above)
Comment (monsieurzhang) [reply utilities/tuminer/README.md]: just updated

File: utilities/tuminer/README.md (hunk 2 above)
Comment (monsieurzhang) [reply utilities/tuminer/README.md]: just updated

**Pull request merged**